cmd/cluster/describe_cluster.go

```diff
@@ -54,11 +55,7 @@ var describeClusterCmd = &cobra.Command{
 			fullClusterContext.Write()
 			return
 		}
-		clustersCtx := formatter.Context{
-			Output: os.Stdout,
-			Format: formatter.NewClusterFormat(viper.GetString("output")),
-		}
-		formatter.ClusterWrite(clustersCtx, resp.GetData())
+		fmt.Println("No cluster found")
```

Comment:
I don't think this is right. This case handles the cases where the output is NOT table. I believe if you do `cluster describe --cluster-name valid-cluster -o json` - this will no longer print anything.

Reply:
Fixed, thanks